src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -217,7 +232,14 @@ public int FrameRate
             }
             set
             {
-                SetValue(FrameRateProperty, value);
+                if (defaultLottieView != null)
+                {
+                    Tizen.Log.Error("NUI", "Cannot set the frame rate to Lottie Animation. If you want to control it, please set `ImageArray` together.\n");
+                }
+                else
+                {
+                    SetValue(FrameRateProperty, value);
+                }
```

Comment:
lottie view를 사용하더라도 FrameRate설정은 그대로 해야 합니다.
그래야지 나중에 ImageArray보다 FrameRate를 설정한 경우에도 FrameRate 설정이 누락되지 않고 적용될 수 있습니다.
``` c#
 loading.FrameRate = 20; 
 loading.ImageArray = new string[] { ...... };
```
```suggestion
                if (defaultLottieView != null)
                {
                    Tizen.Log.Error("NUI", "Cannot set the frame rate to Lottie Animation. If you want to control it, please set `ImageArray` together.\n");
                }
                SetValue(FrameRateProperty, value);
```

Reply:
넵! 수정하겠습니다!
감사합니다.